MIT.md

```diff
@@ -0,0 +1,8 @@
+## Copyright 2021, Amedzro Emmanuel Elikplim
```

Comment:
- [ ] [OPTIONAL]: By convention, license files are usually named simply `LICENSE`. Github provides an easy way to add them to your project, you can read more about it [here](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository).
- [ ] [OPTIONAL]: Consider changing it to 2022 🎆

Reply:
`LICENSE` file is added and a pull request opened